ada_feeding_perception/ada_feeding_perception/segment_from_point.py

```diff
@@ -287,6 +345,10 @@ def segment_image(
             # the seed point
             # TODO: Thorughly test this, in case we need to add more mask cleaning!
             cleaned_mask = get_connected_component(mask, seed_point)
```

Comment:
Just recording a thought: consider erosion/dilation to remove small imperfections in the mask.

Reply:
Added it as an in-text comment.